packages/peregrine/lib/talons/SearchBar/useSuggestedCategory.js

```diff
@@ -1,13 +1,16 @@
 import { useCallback } from 'react';
-
+import { DELIMETER } from '../FilterModal/helpers';
 // TODO: derive from store config when available
-const getLocation = (searchValue, categoryId) => {
+const getLocation = (searchValue, categoryId, label) => {
     // start with the current uri
     const uri = new URL('/search.html', window.location);
 
     // update the query params
     uri.searchParams.set('query', searchValue);
-    uri.searchParams.set('category', categoryId);
+    uri.searchParams.set(
+        'category_id[filter]',
+        `${label}${DELIMETER}${categoryId}`
+    );
```

Comment:
Now that the search page is also relying on the filter modal for the category state, this is tightly coupled to the format expected by the filter components/state. The more I worked on this the more I wished we had a single context for interacting with it. I know that the true state is supposed to be held in the URL but I feel like maybe they should be synchronized.